app/EmptyWikibaseNotification.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class EmptyWikibaseNotification extends Model
+{
+    use HasFactory;
+
+    const FIELDS = [
+        'domain',
+        'first_edited',
```

Comment:
This data is already present in `WikiLifecycleEvents`. What's the reason for choosing to store it again in a different model?